test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -576,9 +582,40 @@ private void DoWorkspaceOpen(string filePath)
 
         }
 
-        private static string ConvertCurrentWorkspaceViewToJsonAndSave(DynamoViewModel viewModel, string filePathBase)
-        {
+        private static string SaveJsonTempWithFolderStructure(DynamoViewModel viewModel, string originalPath, JObject jo)
+        {   
+            // Get all folder structure following "\\test"
+            var expectedStructure = originalPath.Split(new string[] { "\\test" }, StringSplitOptions.None).Last();
+
```

Comment:
Stole this from @QilongTang because it was cleaner than my original implementation :)